### PR TITLE
Working Days

### PR DESCRIPTION
**Pull request by Heavy32**
Upgrade workingDays

### CODE REVIEW

File: WorkDays/WorkDays.cs [hunk 1]
```diff
@@ -11,9 +11,33 @@ public class WorkDays
...
+                {
+                    firstWeekWorkDays = (5 - (int)current.DayOfWeek + 1);
+                    total = firstWeekWorkDays + 2 + (workDays - firstWeekWorkDays);
+                    return total;
+                }
+                else
+                {
+                    return workDays;
+                }
+
```
Comment (alexzaitzev) [inline WorkDays/WorkDays.cs]: Лишняя пустая строка :)

**Review by alexzaitzev — CHANGES_REQUESTED** [review]

File: WorkDays/WorkDays.cs [hunk 2]
```diff
@@ -11,9 +11,33 @@ public class WorkDays
         /// </summary>
         public int DaysCounter(DateTime current, int workDays)
```
Comment (alexzaitzev) [inline WorkDays/WorkDays.cs:11]: Need more tests

File: WorkDays/WorkDays.cs (hunk 1 above)
Comment (Heavy32) [reply WorkDays/WorkDays.cs]: done... Many years ago)

File: WorkDays/WorkDays.cs (hunk 2 above)
Comment (Heavy32) [reply WorkDays/WorkDays.cs:11]: done

File: WorkDays/WorkDays.cs [hunk 3]
```diff
@@ -10,32 +10,10 @@ public class WorkDays
         /// Вывести количество календарных дней, которое потребуется для её выполнения.
         /// </summary>
         public int DaysCounter(DateTime current, int workDays)
-        {
-            int firstWeekWorkDays;
+            => (workDays <= 5) ? (((int)current.DayOfWeek + workDays - 1 > 5) ? (5 - (int)current.DayOfWeek + 1) + 2 + (workDays - (5 - (int)current.DayOfWeek + 1)) : workDays)
```
Comment (alexzaitzev) [inline WorkDays/WorkDays.cs]: (int)current.DayOfWeek - can be extracted to be shortrer

**Review by alexzaitzev — APPROVED** [review]

File: WorkDays/WorkDays.cs (hunk 3 above)
Comment (Heavy32) [reply WorkDays/WorkDays.cs]: done

**Pull request merged**